tests/data/FactoryUtils.ts

```diff
@@ -8,8 +8,9 @@ export default class FactoryUtils {
     return values[i];
   }
 
-  public static getRandomNumber(range: number, minimum = 0, scale = 1): number {
-    return (Math.floor(Math.random() * range) % scale) + minimum;
+  public static getRandomNumber(min: number, max: number, scale = 1): number {
+    const range = Math.floor(Math.random() * (max - min + 1)) + min;
+    return Math.floor(range / scale) * scale;
```

Comment:
```suggestion
    return (range % scale) * scale;
```

Reply:
I'm not sure these are doing the same thing (with the new min, max, scale thing). Let's say we want a value between 30 and 120 with a scale of 30. `range` will give us a value between 30 and 120 that has scale of 1 (e.g. 68), and current line 13 will floor this value to the nearest `scale` (so 68 -> 60, 56 -> 30, 90 -> 90).

New line 13 would do: `(range % scale) * scale` = (68 % 30) * 30 = 8 * 30 = 240 which isn't in the range